sbnanaobj/StandardRecord/SRFlashMatch.h

```diff
@@ -18,9 +20,16 @@ namespace caf
       virtual ~SRFlashMatch();
 
       bool  present;
-      float score;
       float time;
-      float pe;
+      float charge_q;
+      float light_pe;
+      float score;
+      float scr_y;
+      float scr_z;
+      float scr_rr;
+      float scr_ratio;
```

Comment:
Probably my ignorance, but I don't know what `scr` is, could you add a comment? Depending on the answer it might be worth having a little SCR subclass for these four variables (i.e. `scr.y` etc).

In general, I don't think we use underscores elsewhere in StandardRecord, so I'd like to find a way to get rid of them for `charge_q` and `light_pe` too.

Reply:
They are the individual scores for each metric. Sure they could be within a `struct`.

About the underscores, would `chargeq` or `chargeQ` be better?

Reply:
ah. Can you spell the word out in full? And then `score` -> `score.tot` or `score.overall` or something.

We're also mostly avoiding capitals in SR (expect for (E)nergy). The theory is it makes it easier if you don't have to guess the capitalization, but lowercase energy just looks weird. Can you check if there are capital Qs for charge already around? If so, Q is fine, otherwise go for q.

Reply:
I'm now thinking that we could have three structs: `charge, light, score`. We would have `present` and `time` still on their own.

Then, say `charge` would have `charge.q, charge.center`, similar for `light`. 

For `score`: `score.total, score.y, score.z,...`

Reply:
I like it